src/Utils/Transport.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+declare(strict_types=1);
+
+namespace Google\Generator\Utils;
+
+class Transport
+{
+    // Consts are used instead of booleans, because there more transport options may be
+    // supported in the future (e.g. gRPC only).
+    public const GRPC_REST = 1;
+    public const REST_ONLY = 2;
```

Comment:
Why not just REST? Such that the constant names match the command like args and the cross-lang  DIREGAPIC spec in general?

Probably also add `GRPC`  value for completeness, even if it is not supported by the generator (but still can be used by the validation logic).

Reply:
Done, leaving out `GRPC` since it's not used anywhere.

Technically, `REST` also isn't used anywhere and this could be a bool. I initially tried that out, but the ensuing logic (e.g. `isGrpcRest()`) implied that the negative state had only one value, which looked suboptimal from a readability and maintenance perspective.